sprint8/exercise2.js

```diff
@@ -0,0 +1,15 @@
+// folosesc functia "calculateSum(numbers)" shi insushesc un array de numere ca parametru.
+function calculateSum(numbers) {
+// Adaug metoda "reduce()" care itereaza prin fiecare element din lista, shi ii face suma.
+//"accumulator" suma pana la moment, iar "currentValue va fi elementul curent din array"; 
+// accumulator + currentValue: Aceasta aduna valoarea curenta la suma acumulata; "0" este valoarea initiala a accumulator-ului.
+    return numbers.reduce((accumulator, current) => accumulator + currentcurrentValue, 0);
```

Comment:
```suggestion
    return numbers.reduce((accumulator, current) => accumulator + current, 0);
```
ai o eroare de sintaxa